apprise/plugins/NotifyNtfy.py

```diff
@@ -0,0 +1,573 @@
+            self.priority = priority
+
+        if self.priority not in NTFY_PRIORITIES:
+            msg = 'An invalid Ntfy Priority ({}) was specified.'.format(
+                priority)
+            self.logger.warning(msg)
+            raise TypeError(msg)
+
+        # Any optional tags to attach to the notification
+        self.__tags = parse_list(tags)
+
+        # Build list of topics
+        topics = parse_list(targets)
+        self.topics = []
+        for _topic in topics:
+            topic = validate_regex(
+                _topic, *self.template_tokens['topic']['regex'])
+            if not topic:
+                self.logger.warning(
+                    'A specified Ntfy topic ({}) is invalid and will be '
+                    'ignored'.format(_topic))
+                continue
+            self.topics.append(topic)
+        return
+
+    def send(self, body, title='', notify_type=NotifyType.INFO, **kwargs):
+        """
+        Perform Ntfy Notification
+        """
+
+        # error tracking (used for function return)
+        has_error = False
+
+        if not len(self.topics):
+            # We have nothing to notify; we're done
+            self.logger.warning('There are no Ntfy topics to notify')
+            return False
+
+        # Prepare our headers
+        headers = {
+            'User-Agent': self.app_id,
+        }
+
+        if self.priority != NtfyPriority.NORMAL:
+            headers['X-Priority'] = self.priority
+
+        if title:
+            headers['X-Title'] = title
+
+        if self.attach is not None:
+            headers['X-Attach'] = self.attach
+
+        if self.click is not None:
+            headers['X-Click'] = self.click
+
+        if self.delay is not None:
+            headers['X-Delay'] = self.delay
+
+        if self.email is not None:
+            headers['X-Email'] = self.email
+
+        if self.__tags:
+            headers['X-Tags'] = ",".join(self.__tags)
+
+        # Prepare our payload
+        payload = body
+
+        auth = None
+        if self.mode == NtfyMode.CLOUD:
+            # Cloud Service
+            template_url = self.cloud_notify_url
+
+        else:  # NotifyNtfy.PRVATE
+            # Allow more settings to be applied now
+            if self.user:
+                auth = (self.user, self.password)
+
+            # Prepare our Ntfy Template URL
+            schema = 'https' if self.secure else 'http'
+
+            template_url = '%s://%s' % (schema, self.host)
+            if isinstance(self.port, int):
+                template_url += ':%d' % self.port
+
+        template_url += '/{topic}'
+
+        # Create a copy of the subreddits list
+        topics = list(self.topics)
+        while len(topics) > 0:
+            # Retrieve our topic
+            topic = topics.pop()
+
+            # Create our Posting URL per topic provided
+            url = template_url.format(topic=topic)
+            self.logger.debug('Ntfy POST URL: %s (cert_verify=%r)' % (
```

Comment:
I know this may seem nitpicky, but can you change all of the `Ntfy` to `ntfy` in this PR? It bugs me so much. It's so irrational :-D

Reply:
Done